source/renderer/app/components/wallet/WalletSendForm.js

```diff
@@ -299,8 +301,15 @@ export default class WalletSendForm extends Component<Props, State> {
     return receiverField.value.length > 0;
   };
 
-  hasAssetValue = (asset: Field) => {
```

Comment:
@aleksandardjordjeviciohk what happened with this? Why was it removed?

Reply:
Not being used at this component, obsolete method.